.gitignore

```diff
@@ -4,4 +4,5 @@ node_modules
 build
 # The output of the singleModule example
 examples/singleModule/templates/index.js
-*.log
\ No newline at end of file
+*.log
+.idea/
```

Comment:
Throw this in your global `.gitignore` instead of adding it to repos you contribute to: https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore